op-verifier/deploy_l1/10_testl1.ts

```diff
@@ -10,7 +10,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const OPVerifier = await deployments.get('OPVerifier');
   const TestL2 = await hre.companionNetworks['l2'].deployments.get('TestL2');
-
+  console.log([OPVerifier.address, TestL2.address])
```

Comment:
Can you remove this, or if it's needed, change it to include text explaining what the addresses are?